framework/ui/src/main/java/com/google/android/catalog/framework/ui/CatalogScreen.kt

```diff
@@ -45,10 +49,11 @@ import com.google.android.catalog.framework.ui.components.SearchTopAppBar
 
 internal const val CATALOG_DESTINATION = "catalog"
 
-@OptIn(ExperimentalMaterial3Api::class)
+@OptIn(ExperimentalMaterial3Api::class, ExperimentalAnimationApi::class)
```

Comment:
nit: `ExperimentalAnimationApi` seems unused.

Reply:
It happened when I split the PR. The next PR uses that so I will merge